tests/JavaTest.java

```diff
@@ -1224,4 +1226,11 @@ static void TestVectorOfBytes() {
             System.exit(1);
         }
     }
+
+    static void TestFail(String message) {
```

Comment:
I know this is for the future fix, but we shouldn't include the code in this PR.

Reply:
I just don't think we should add code that isn't referenced by anything in this PR. I know it is pointed to by a TODO, but I think when that TODO is completed, the code can be added. Its more atomic that way.